src/lib/storage/table.hpp

```diff
@@ -87,5 +88,8 @@ class Table : private Noncopyable {
   uint32_t _max_chunk_size;
 
   void _append_new_chunk();
+
+  static void compress_segment(std::promise<std::shared_ptr<BaseSegment>> promise, const std::string type,
```

Comment:
leading underscore (does lint let this pass? Are no leading underscores required for static methods?)

Reply:
Funny, the linter passes, yet CONTRIBUTING.md says it should have a leading underscore.